lib/ssz.ex

```diff
@@ -162,13 +164,9 @@ defmodule Ssz do
     function_exported?(module, function, arity)
   end
 
-  @spec encode_u256(non_neg_integer) :: binary
-  def encode_u256(num) do
-    num
-    |> :binary.encode_unsigned(:little)
-    |> String.pad_trailing(32, <<0>>)
-  end
+  @spec encode_u256(Types.uint256()) :: Types.bytes32()
+  def encode_u256(num) when num < @max_u256, do: <<num::little-size(256)>>
```

Comment:
You can use `<=` if you already did the `-1`. If you still want to use the `<`, then you can remove the `-1` from above

Reply:
Good catch!